### PR TITLE
Fix #2400 : Resolved ktlint max line in administratorcontrols test package

### LINKED ISSUE #2400: Resolve ktlint max line in administratorcontrols test package
**Describe the bug**
We are preventing the 100 char length at some places like test case fun name. 
`// ktlint-disable max-line-length`

**Expected behavior**
We need to remove this prevention and fix the function name under 100 char length.

### PR DESCRIPTION
**Pull request by coder2699**
## Explanation

Fixes #2400 : Removed ' // ktlint-disable max-line-length ' and fixed the functions name under 100 char length.
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by coder2699** [comment]
@anandwana001 I am fixing the failed tests.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 1]
```diff
@@ -304,7 +304,7 @@ class AdministratorControlsActivityTest {
   }
 
   @Test
-  fun testAdministratorControlsFragment_loadFragment_topicUpdateOnWifiSwitchIsNotChecked_autoUpdateTopicSwitchIsNotChecked() { // ktlint-disable max-line-length
+  fun testAdministratorControlsFragment_topicUpdateWifiSwitchUncheck_autoUpdateTopicSwitchUncheck() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]:
```kotlin
fun testAdministratorControlsFragment_wifiSwitchIsUncheck_autoUpdateSwitchIsUncheck()
```

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 2]
```diff
@@ -335,7 +335,7 @@ class AdministratorControlsActivityTest {
   }
 
   @Test
-  fun testAdministratorControlsFragment_topicUpdateOnWifiSwitchIsChecked_configurationChange_checkIfSwitchIsChecked() { // ktlint-disable max-line-length
+  fun testAdministratorControlsFragment_topicUpdateOnWifiSwitchChecked_configChange_switchIsChecked() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]:
```kotlin
fun testAdministratorControlsFragment_clickWifiSwitch_configChange_wifiSwitchIsChecked()
```

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 3]
```diff
@@ -412,7 +412,7 @@ class AdministratorControlsActivityTest {
   }
 
   @Test
-  fun testAdministratorControlsFragment_loadFragment_clickEditProfile_checkOpensProfileListActivity() { // ktlint-disable max-line-length
+  fun testAdministratorControlsFragment_loadFragment_clickEditProfile_opensProfileListActivity() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]:
```kotlin
fun testAdministratorControlsFragment_clickEditProfile_opensProfileListActivity()
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @coder2699 
left a few comments. 

Also please remove all the `loadFragment`, it doesn't make any sense in the test case. 

Also, update your branch with the latest develop

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: there are only 2 switches `wifi` and `autoUpdate` so no need of `topicUpdate`

**Comment by coder2699** [comment]
> Thanks @coder2699
> left a few comments.
> 
> Also please remove all the `loadFragment`, it doesn't make any sense in the test case.
> 
> Also, update your branch with the latest develop

@anandwana001 Ok, I will do as mentioned

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 3 above)
Comment (coder2699) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Thanks. Done.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 2 above)
Comment (coder2699) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Thanks. Done.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 1 above)
Comment (coder2699) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Thanks. Done.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Could you please correct all the loadFragment too?

**Comment by coder2699** [comment]
> Could you please correct all the loadFragment too?

@anandwana001 is it fine now?

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 4]
```diff
@@ -138,7 +138,7 @@ class AdministratorControlsActivityTest {
   }
 
   @Test
-  fun testAdministratorControlsFragment_loadFragment_displayGeneralAndProfileManagement() {
+  fun testAdministratorControlsFragment_displayGeneralAndProfileManagement() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: `testAdministratorControlsFragment_generalAndProfileManagementIsDisplayed()`

**Review by anandwana001 — COMMENTED** [review]
Same with other 3, IsDisplayed sounds good as it gives a sense of testing.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 4 above)
Comment (coder2699) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Thanks for review sir, I will do so.

Comment (coder2699) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Done, PTAL.

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @coder2699

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**